sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/src/main/java/com/azure/spring/cloud/config/implementation/State.java

```diff
@@ -13,25 +13,25 @@ class State {
 
     private final Instant nextRefreshCheck;
 
-    private final String key;
+    private final String originEndpoint;
```

Comment:
Why changing from `key` to `originEndpoint`?

Reply:
The value of the key was always the endpoint or endpoint + "_featureflag", this is to clarify that it needs to be the originEndpoint not just any endpoint.